tests/kafkatest/tests/core/compatibility_test_new_broker_test.py

```diff
@@ -46,12 +45,12 @@ def setUp(self):
 
     @cluster(num_nodes=6)
     @parametrize(producer_version=str(LATEST_0_8_2), consumer_version=str(LATEST_0_8_2), compression_types=["none"], new_consumer=False, timestamp_type=None)
-    @parametrize(producer_version=str(LATEST_0_8_2), consumer_version=str(LATEST_0_9), compression_types=["none"], new_consumer=False, timestamp_type=None)
```

Comment:
I'm a fan of removing unneeded parameterizations, but is this ok to remove just because we don't think there's anything between 0.8 and 0.9 clients that would be affected by running against an 0.10 broker?